App.js

```diff
@@ -47,8 +48,29 @@ class Content extends React.Component {
         this.setState({ userInfo: temp })
     }
 
-    async handleLogin(username1, password) {
-        this.setState({ home: true, authentication: "", username: username1, homeText: "SHS-chatBot" })
+    async handleLogin() {
```

Comment:
This looks great! Will we update the avatar of the user here or it will just be a photo uploaded?

Reply:
Photo will be uploaded. Still need to finish up a bit on the PR. I'll close it by EOD